src/devices/Adxl357/README.md

```diff
@@ -0,0 +1,35 @@
+# ADXL357 - Accelerometer
+
+ADXL357 is a 3-Axis digital accelerometer 40g with 20-bit resolution measurement at up to ±40g.
+Sensitivity is configurable (±10g, ±20g, ±40g).
+Has a built in temperature sensor.
+
+## Sensor Image
+
+![Source: https://wiki.seeedstudio.com/Grove-3-Axis_Digital_Accelerometer_40g-ADXL357/](sensor.png)
+Source: https://wiki.seeedstudio.com/Grove-3-Axis_Digital_Accelerometer_40g-ADXL357/
+
+## Usage
+
+```C#
```

Comment:
in pure theory, it shuold be csharp and no c#, now GitHub is nice enough to display both correctly ;-)

Reply:
@Ellerbach Agree with all of it and changed code accordingly.